### PR TITLE
Add placeholders for cookie privacy notice

### PR DESCRIPTION
**Pull request by joescottdave**
- Add cookie route and controller
- Add placeholder views for cookie policy

Still to be added: a global link in the footer(?)

Not sure how we like '/cookie' as a route name or whether 'Privacy notice/policy' sounds more appropriate.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@
   resources :landing, only: [:index]
 
   get 'landing/hpi', to: 'landing#hpi'
+  get 'cookie', to: 'cookie#index'
```
Comment (ijdickinson) [inline config/routes.rb:10]: I think I'd go with a plural "cookies" over "cookie" here, but per your question `/privacy-policy` or `/cookies-policy` might be better.

File: config/routes.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
   resources :landing, only: [:index]
 
   get 'landing/hpi', to: 'landing#hpi'
+  get 'privacy-notice', to: 'privacy#index'
```
Comment (joescottdave) [inline config/routes.rb:10]: I didn't know whether to add either a leading 'landing/' or 'doc/' for this route, it's not immediately clear to me what the logical criteria is/was for them both, and whether the privacy policy should be grouped with one, the other, or standalone like this.

Comment (ijdickinson) [reply config/routes.rb:10]: I suspect the `landing/hpi` route is never used any more. HPI was the fore-runner to UKHPI, which got a complete ground-up rewrite when we migrated to the country-wide index. I could, however, see an argument for putting `doc/privacy-notice` alongside `doc/accessibility`.

Comment (joescottdave) [reply config/routes.rb:10]: I think that seems correct, for consistency: `doc/privacy-notice`.

Should the `landing/hpi` route be removed?

Comment (ijdickinson) [reply config/routes.rb]: If memory serves, the `landing/hpi` route does a redirect to UKHPI, so I think it should stay.

**Comment by joescottdave** [comment]
Task is paused awaiting Welsh translation.

**Review by ijdickinson — COMMENTED** [review]
I don't think you need to add a new controller. If you make the route:

```ruby
  get 'doc/privacy-notice', to: 'doc#privacy_notice'
```

you can rename `app/views/privacy/index.html.erb` to `app/views/doc/privacy_notice.html.erb`. Then we would be using the same pattern for privacy and accessibility.

**Comment by joescottdave** [comment]
I've made the changes to file/route organisation that you suggested @ijdickinson

**Review by ijdickinson — APPROVED** [review]
lgtm

**Pull request merged**